addons/binding/org.openhab.binding.vallox/src/main/java/org/openhab/binding/vallox/internal/serial/Telegram.java

```diff
@@ -0,0 +1,363 @@
+package org.openhab.binding.vallox.internal.serial;
```

Comment:
Every Java file must have a license header. You can run `mvn license:format` on the root of the repo to automatically add missing headers.